cogs/qrchecker/commandsCore.py

```diff
@@ -0,0 +1,21 @@
+from redbot.core.commands import Context
+
+from .constants import KEY_ENABLED
+from .core import Core
+
+
+class CommandsCore(Core):
+    async def cmdQrCheckerToggle(self, ctx: Context):
+        """Toggle QR code checking"""
+        guild = ctx.guild
+        if not guild:
+            return
```

Comment:
I know this was in the previous code, but could you also remove this unnecessary return in a subsequent refactoring PR?

Reply:
Yes. We can also have an `assert` here as a sanity check while reading the code that prior to this method call, `Context` certainly comes from a guild message/interaction.